### DISCUSSION TITLE
where can i find the roadmap for trino project?

### DISCUSSION THREAD

**Discussion by johndb2016** [trinodb/trino#13724] [Q&A]
i wanna to know what kinds of new features are coming, so can someone tells me where is the roadmap? The lables "roadmap" are too distracted, is there a official doc to record the roadmap?

**Comment by sunriseLe** [comment]
same question，need your help~

**Comment by raunaqmorarka** [comment]
The best way to find out is to join the community slack https://trino.io/slack.html and ask questions there about the specific improvements that you're interested in.

**Comment by sunriseLe** [comment]
你好，我是李晓芳，我已收到你的邮件，我会定时查看的。欢迎经常联系
